pom.xml

```diff
@@ -30,7 +30,7 @@
     <module>okhttp-tls</module>
     <module>okhttp-urlconnection</module>
 
-    <module>okhttp-logging-interceptor</module>
+    <module>okhttp-logging</module>
```

Comment:
This and the is a public API change, it makes sense cleanup wise.  But it may be simpler to leave in the current location.  It also makes it possible for two versions of the existing logger to be on the classpath at the same time, because both maven/gradle dependencies can be included.

Reply:
This change **requires** a redirection pom

Reply:
Let me know if you prefer me to simply revert the POM change or add a redirection POM.